src/PlusImageProcessing/vtkPlusBoneEnhancer.cxx

```diff
@@ -57,6 +57,8 @@ vtkPlusBoneEnhancer::vtkPlusBoneEnhancer()
   ProcessedLinesImage(NULL),
   FirstFrame(true)
 {
+  this->SetSaveIntermediateResults(false);

```

Comment:
Please initialize this variable a few lines above, like the other member variables.